docs/topics/what-is-the-toolkit.adoc

```diff
@@ -37,5 +41,5 @@ ifndef::getting-started-guide[]
 [discrete]
 == How do I learn more?
 
-See the link:{ProductDocIntroToMTAGuideURL}[_{IntroToMTABookName}_] to learn more about the features, supported configurations, system requirements, and available tools in the {ProductName}.
+See the link:{ProductDocIntroTo{ProductShortName}GuideURL}[_{IntroTo{ProductShortName}BookName}_] to learn more about the features, supported configurations, system requirements, and available tools in the {ProductName}.

```

Comment:
Does this render correctly?

Reply:
@RichardHoch "{ProductShortName} for developers"

Reply:
Then write out the text.